ideascube/conf/base.py

```diff
@@ -105,27 +105,15 @@
 
 USE_TZ = True
 
-# Languages that will be available in all forms.
+# Languages that will be available as UI translations
 AVAILABLE_LANGUAGES = (
```

Comment:
I think at this point we could just use `LANGUAGES`, also removing the postprocess in `settings.py`.